projects/core/src/state/utils/entity-loader/entity-loader.action.ts

```diff
@@ -102,12 +104,9 @@ export class EntityFailAction implements EntityLoaderAction {
    *             the feature toggle `ssrStrictErrorHandlingForHttpAndNgrx`.
    */
   constructor(entityType: string, id: EntityId);
-  constructor(
-    entityType: string,
-    id: EntityId,
-    public error?: any
-  ) {
+  constructor(entityType: string, id: EntityId, error?: any) {
     this.meta = entityFailMeta(entityType, id, error);
+    this.error = error;
```

Comment:
[note]
Here it's OK, to implement this line. Thanks to it, all actions extending this class won't need to implement analogical mapping for `this.error` property, I guess.

Same for: `LoaderFailAction` and `EntityScopedFailAction`